.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release a new version
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    if: "!github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.0.7
+
+      - name: Deploy artifacts and notify on Slack
+        env:
+          AZURE_STORAGE_CONNECTION_STRING: ${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}
```

Comment:
Still using azure? Thought we were moving out

Reply:
Yep, we need first to ask Bintray to sync our artifacts on JCenter. Once sync is automated, we can move forward on JCenter and drop Azure. (See EE-1167)